Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -417,6 +424,11 @@ public static Furniture PlaceInstance(Furniture proto, Tile tile)
     /// <param name="deltaTime">The time since the last update was called.</param>
     public void Update(float deltaTime)
     {
+        if (isBeingDestroyed)
```

Comment:
This could prevent a machine from working while it will be destroyed, like a door not opening. We could probably just prevent new jobs being added, but let the machines work until they get destroyed.